### ISSUE TITLE
Unable to issue/renew certificate anymore

### ISSUE THREAD

**Issue by hostalp** [rkosegi/cert-manager-webhook-active24#124]
Recently I started having issues with the certificate renewal (worked a few months ago, but not any longer).

```
I0809 13:34:52.061900       1 main.go:78] Present: fqdn=_acme-challenge.somedomain.net., zone=somedomain.net., key=qUXCrf7evXjIQfBWEItLIwuCxzoDQCOpK5aRitCSMpc
I0809 13:34:52.061924       1 main.go:189] recordName: ResolvedZone=somedomain.net., ResolvedFQDN=_acme-challenge.somedomain.net.
I0809 13:34:52.066250       1 active24.go:37] FindTxtRecord: name=_acme-challenge, text=qUXCrf7evXjIQfBWEItLIwuCxzoDQCOpK5aRitCSMpc
E0809 13:34:52.295304       1 active24.go:41] "invalid API response" err="invalid response from api: 404" code="404 404"
```

* Deployed using Helm chart v1.0.1 with image set to `ghcr.io/rkosegi/cert-manager-webhook-active24:v1.0.1` (you've got it wrong in the v1.0.1 - it points to `rkosegi/cert-manager-webhook-active24:1.0.1` which doesn't exist).
* API key is correct (issued last year, still valid)
* ClusterIssuer also appears correct (created last year, not changed since then)

Do you know if the Active24 API hasn't changed in any way?
I looked at https://api.active24.com/swagger-ui.html and tried some manual requests to get some records (using my API key) - those work and it appears to match what the https://github.com/rkosegi/active24-go/blob/main/active24/dns.go does. But maybe I'm missing something.

Note that I can't find the expected TXT record there - based on the above piece of log it should be at least created, but there's nothing visible in those records. Maybe even that part doesn't work properly, or it's just a consequence.

**Comment by rkosegi** [comment]
Petr, thanks for bug report. 
I can confirm issue. Not sure about root cause, I'm sure this worked before.
Even stranger is what you pointed out already, swagger UI or curl are producing OK results while pretty same requests coming from this app not. Let me do more tests

**Comment by rkosegi** [comment]
This should be fixed in v1.0.2
